elasticsearch/client.go

```diff
@@ -106,13 +108,33 @@ func (cli *Client) AddDocument(ctx context.Context, indexName, documentType, doc
 
 }
 
+// BulkUpdate uses an HTTP post request to submit data to Elastic Search
+func (cli *Client) BulkUpdate(
+	ctx context.Context, esDestIndex string, esDestURL string, bulk []byte) ([]byte, int, error) {
+
+	uri := fmt.Sprintf("%s/%s/_bulk", esDestURL, esDestIndex)
+	jsonBody, status, err := cli.callElastic(ctx, uri, "POST", bulk)
+
+	if err != nil {
+		log.Info(ctx, "error posting bulk request", log.Data{
+			"err": err})
```

Comment:
formatting

Reply:
Um this is a duplicate log for the correctly represented log on line 121. Can lines 119 and 120 be deleted.